lib/modules/manager/types.ts

```diff
@@ -22,9 +23,7 @@ export interface ExtractConfig {
   skipInstalls?: boolean;
 }
 
-export interface CustomExtractConfig
-  extends ExtractConfig,
-    RegexManagerTemplates {
+export interface CustomExtractConfig extends RegexManagerTemplates {
```

Comment:
`CustomExtractConfig` is only used in manager `regex` and no fields of `ExtractConfig`  are used there, so this is a correction.